docs/dynamic-plugins/versions.md

```diff
@@ -1,17 +1,17 @@
 # Version Compatibility Matrix
 
-## RHDH 1.4 (pre-release, versions can change for final release)
+## RHDH 1.4
 
 <!-- source
-https://github.com/janus-idp/backstage-showcase/blob/main/backstage.json
+https://github.com/janus-idp/backstage-showcase/blob/release-1.4/backstage.json
 -->
 
-Based on [Backstage 1.31.3](https://backstage.io/docs/releases/v1.31.0)
+Based on [Backstage 1.32.6](https://backstage.io/docs/releases/v1.32.0)
```

Comment:
this is for 1.4 but your changes are targetting `main` branch (1.5), not `release-1.4`.